contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -74,4 +81,15 @@ contract TimedCrowdsale is Crowdsale {
     function _preValidatePurchase(address beneficiary, uint256 weiAmount) internal onlyWhileOpen view {
         super._preValidatePurchase(beneficiary, weiAmount);
     }
+
+    /**
+     * @dev Extend crowdsale
+     * @param newClosingTime Crowdsale closing time
+     */
+    function _extendTime(uint256 newClosingTime) internal {
+        require(newClosingTime > _closingTime);
```

Comment:
I think we should also require the crowdsale to not have yet closed: we're otherwise allowing for a crowdsale to be re-opened after closing.

Reply:
I think in most cases funders may decide to extend ICO after it was ended. Maybe to deny this behavior for `RefundableCrowdsale`?

Reply:
Or introduce separate method `function _reopenAndExtendTime(uint256 newClosingTime) internal`?

Reply:
`RefundableCrowdsale`, `PostdeliveryCrowdsale`, etc. would all have issues with this, it feels like we'd be opening a huge can of worms. @frangio WDYT?

Reply:
Good catch @nventuro... The rest of the crowdsale suite definitely assumes that crowdsales are never reopened. We should disallow it.